### ISSUE TITLE
Sys.getenv("SHINYPROXY_USERGROUPS")) returns empty string

### ISSUE THREAD

**Issue by panoptikum** [openanalytics/shinyproxy#219]
Hello,

I was trying to condition some part of my app on certain user groups, however if try to retrieve the user groups of the logged in user I only get an empty object:

`Sys.getenv("SHINYPROXY_USERGROUPS"))`

If I try to access the user via:

`print(Sys.getenv("SHINYPROXY_USERNAME"))
`
it works perfectly.

To give a reproducible example, my application.yml looks as follows:

```
proxy:
  port: 8081
  authentication: simple
  admin-groups: admins
  container-log-path: home/felix/container-logs
  users:
  - name: jack
    password: password
    groups: admins
  - name: jeff
    password: password
    groups: pv_qs
  - name: jane
    password: password
    groups: [management, test]
  docker:
      internal-networking: true
  specs:
  - id: 01_hello
    display-name: Hello Application
    description: Application which demonstrates the basics of a Shiny app
    container-cmd: ["R", "-e", "shinyproxy::run_01_hello()"]
    container-image: openanalytics/shinyproxy-demo
    container-network: sp-example-net
  - id: 06_tabsets
    container-cmd: ["R", "-e", "shinyproxy::run_06_tabsets()"]
    container-image: openanalytics/shinyproxy-demo
    container-network: sp-example-net
  - id: 02_lifecycle
    display-name: Lifecycle
    container-cmd:  ["R", "-e", "shiny::runApp('/root/lifecycle')"]
    container-image: lifecycle
    container-network: sp-example-net

logging:
  file:
    shinyproxy.log
```
Is this a known bug? Any hints for a solution?

Much appreciated.

**Comment by panoptikum** [comment]
I have to correct myself.

It only returns nothing, if I assign at least two groups to the user like I did for Jane.

Another "bug" I have discovered:

When I write "management", everything in small letters, I get in shiny "MANAGEMENT" everything in capital letters. This is very misleading, if condition on this value and if this is not anywhere mentioned in your doc.

**Comment by panoptikum** [comment]
Ok, I see that in contrast to the specification of access-groups, I have to separate groups only with a comma. Well, an example in your docs would be good for this...
But the uppercase error persists and is not happening for the user name.

**Comment by tverbeke** [comment]
Thanks for the feedback, @panoptikum. We will look into it.

**Comment by panoptikum** [comment]
Thanks.

I find it confusing that I have to specify groups for a user as follows:

groups = engineers, scientists

while I specify access-groups for an app as:

access-groups = [engineer, scientists]

Maybe this way to specify groups for a user is incorrect, but [engineers, scientists] does not work.

For clarity: I use shinyproxy 2.3.3

**Comment by LEDfan** [comment]
Hi, I just released ShinyProxy 3.0.1 which fixes this. 
You can now use array notation for the `groups` of a user.
